lambda_functions/submission_service/submit_item.py

```diff
@@ -49,37 +54,41 @@ def submit_item(event, context, is_test=False, session=None):
 
         try:
             # Item already exists, item_id in submission is the id of the found item
-            found_item = item_handler.get_item_by_content(
+            item = item_handler.get_item_by_content(
                 content, is_test, session)
-            submission.item_id = found_item.id
+            submission.item_id = item.id
             new_item_created = False
 
         except Exception:
             # Item does not exist yet, item_id in submission is the id of the newly created item
-            new_item = Item()
-            new_item.content = content
-            new_item.item_type_id = item_type_id
-            new_item.type = type
-            created_item = item_handler.create_item(
-                new_item, is_test, session)
+            item = Item()
+            item.content = content
+            item.item_type_id = item_type_id
+            item.type = type
+            item = item_handler.create_item(
+                item, is_test, session)
             new_item_created = True
-            submission.item_id = created_item.id
+            submission.item_id = item.id
             stage = os.environ['STAGE']
             client.start_execution(
                 stateMachineArn='arn:aws:states:eu-central-1:891514678401:stateMachine:SearchFactChecks_new-'+stage,
-                name='SFC_' + created_item.id,
+                name='SFC_' + item.id,
                 input="{\"item\":{"
-                "\"id\":\"" + created_item.id + "\","
-                "\"content\":\"" + created_item.content + "\" } }"
+                "\"id\":\"" + item.id + "\","
+                "\"content\":\"" + item.content + "\" } }"
             )
 
         # Create submission
         submission_handler.create_submission_db(submission, is_test, session)
+        try:
+            send_confirmation_mail(submission)
+        except EmailNotificationError:
+            pass
```

Comment:
Why raise the Exception, if it is not handled?

In this case `send_confirmation_mail` could just pass the Exception.